### PR TITLE
fix: fix translaters and broken dependence

### PR DESCRIPTION
**Pull request by rampmaster**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 vendor/
 composer.lock
 phpunit.xml
+.idea
```
Comment (blaugueux) [inline .gitignore]: This should be in your global .gitignore not inside the repo itself.

**Comment by blaugueux** [comment]
Thanks for the translations :1:

File: composer.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
     ],
     "require": {
         "php": ">=5.4.0",
+        "zendframework/zendxml": "^1.0",
```
Comment (blaugueux) [inline composer.json]: Why adding this dependency?

**Comment by Sander-Toonen** [comment]
The extra dependency is not needed. It is already via yzalis/supervisor --> zendframework/zend-xmlrpc --> zendframework/zendxml

Could you please merge the translations?

**Comment by blaugueux** [comment]
@rampmaster Thanks for your work here, translation has been merged here b2a155f10e388ea8e9ea86db01476bdc8d023dfe /cc @Sander-Toonen

**Pull request closed without merging**